api/src/main/java/org/cardanofoundation/rosetta/common/enumeration/NetworkIdentifierType.java

```diff
@@ -3,9 +3,10 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 public enum NetworkIdentifierType {
-  CARDANO_MAINNET_NETWORK("mainnet", 1, 764824073L),
-  CARDANO_TESTNET_NETWORK("testnet", 0, 1097911063L),
-  CARDANO_PREPROD_NETWORK("preprod", 0, 1);
+  CARDANO_MAINNET_NETWORK("Mainnet", 1, 764824073L),
```

Comment:
Why you are using an upper-case here?
Also, we could use Constants here

Reply:
Because of public static final String MAINNET = "Mainnet" in Constants